src/IssueViz.Security/OpenInIDE/Api/IHotspotToIssueVisualizationConverter.cs

```diff
@@ -55,7 +55,7 @@ public IAnalysisIssueVisualization Convert(SonarQubeHotspot sonarQubeHotspot)
 
         private Hotspot ConvertToHotspot(SonarQubeHotspot sonarQubeHotspot)
         {
-            var filePath = absoluteFilePathLocator.Locate(sonarQubeHotspot.ComponentPath);
+            var filePath = absoluteFilePathLocator.Locate(sonarQubeHotspot.ComponentPath) ?? sonarQubeHotspot.ComponentPath;
```

Comment:
Noticed this bug yesterday.
I always set the value to whatever we get back from the locator, which can be null if the file is not found.
We then try to navigate to it, and fail, but the message is not descriptive ("Failed to open file ''", or something of the sort), and there is no file path in the hotspots table.
So if it's null, I just keep the original value.
The navigation will still fail and it will be more descriptive.